WorkInProgress/Administration/xASL_adm_CoregisterAtlas.m

```diff
@@ -10,9 +10,13 @@
 pathTPMAtlasFile = 'mni_icbm152_gm_tal_nlin_asym_09c.nii';
 pathTPMAtlasFileWM = 'mni_icbm152_wm_tal_nlin_asym_09c.nii';
 pathTPMAtlasFileGMWM = 'mni_icbm152_gmwm_tal_nlin_asym_09c.nii';
+pathColinDir = 'mni_colin27_2008';
```

Comment:
We don't save these in ExploreASL right? (at least we shouldn't if we don't use them in the pipeline), should we save them in CustomScripts for ease of use?

Reply:
Haha. No of course not - these are all on my disc in a temp dir. I only save the result to ExploreASL dir.
But it's a good idea to add also the input files to CustomScripts so that this can be re-run easier.